### PR TITLE
doc: Fixes for the changelog

### PR DESCRIPTION
**Pull request by umapraseeda**
Fixes for the changelog
To be merged after https://github.com/nrfconnect/sdk-nrf/pull/7905 

Signed-off-by: Uma Praseeda <uma.praseeda@nordicsemi.no>

### CODE REVIEW

File: doc/nrf/releases/release-notes-changelog.rst [hunk 1]
```diff
@@ -273,11 +275,11 @@ Other libraries
...
 
-* :ref:`caf_ble_state` running on Bluetooth Peripheral no longer automatically sends security request right after Bluetooth LE connection is established.
-  The :kconfig:option:`CONFIG_CAF_BLE_STATE_SECURITY_REQ` can be used to enable this feature.
-  The option can be used for both Bluetooth Peripheral and Bluetooth Central.
+* :ref:`caf_ble_state`:
 
-|no_changes_yet_note|
+  * Running on Bluetooth Peripheral no longer automatically send security request immediately after Bluetooth LE connection is established.
+    The :kconfig:option:`CONFIG_CAF_BLE_STATE_SECURITY_REQ` can be used to enable this feature.
+    The option can be used for both Bluetooth Peripheral and Bluetooth Central.
```
Comment (peknis) [inline doc/nrf/releases/release-notes-changelog.rst:282]:
```suggestion
  * Running on Bluetooth Peripheral no longer automatically sends security request immediately after a Bluetooth LE connection is established.
    Use the :kconfig:option:`CONFIG_CAF_BLE_STATE_SECURITY_REQ` option to enable this feature.
    The option can be used for both Bluetooth Peripheral and Bluetooth Central.
```

File: doc/nrf/releases/release-notes-changelog.rst [hunk 2]
```diff
@@ -373,16 +375,16 @@ Documentation
 
 * Added:
 
-* Added documentation page for the :ref:`lib_flash_map_pm` library.
-* :ref:`ug_thread_prebuilt_libs` as a separate page instead of being part of :ref:`ug_thread_configuring`.
+  * Documentation for the :ref:`lib_flash_map_pm` library.
```
[the remark this answers is not in the document]
Comment (peknis) [reply doc/nrf/releases/release-notes-changelog.rst:378]: If the library has been available already, but without documentation, this entry belongs here.

**Review by peknis — CHANGES_REQUESTED** [review]

File: doc/nrf/releases/release-notes-changelog.rst [hunk 3]
```diff
@@ -139,7 +139,7 @@ Bluetooth samples
 
 * :ref:`ble_nrf_dm` sample:
 
-  * Split the configuration of the Distance Measurement module from the Nordic Distance Measurement library.
+  * Split the configuration of the :ref:`mod_dm` module from the :ref:`nrf_dm`.
```
[the remark this answers is not in the document]
Comment (umapraseeda) [reply doc/nrf/releases/release-notes-changelog.rst:142]: Seems to work locally without the nrfxlib prefix.  And the build error in CI is related to the flash patch library.

**Review by peknis — APPROVED** [review]

File: doc/nrf/releases/release-notes-changelog.rst [hunk 4]
```diff
@@ -299,33 +301,31 @@ Other libraries
...
+* :ref:`doc_fw_info` module:
 
-  * :ref:`doc_fw_info` module:
+  * Fixed a bug where MCUboot would experience a fault when using the :ref:`doc_fw_info_ext_api` feature.
 
-    * Fixed a bug where MCUboot would experience a fault when using the :ref:`doc_fw_info_ext_api` feature.
+* :ref:`lib_nrf_rpc`:
 
-  * :ref:`lib_nrf_rpc`:
+  * These libraries can use different transport implementation for each nRF RPC group.
```
Comment (umapraseeda) [inline doc/nrf/releases/release-notes-changelog.rst:314]: @peknis , please check if this is correct. Changed to libraries as the link is for nRF RPC libraries

Comment (peknis) [reply doc/nrf/releases/release-notes-changelog.rst:314]: Indeed, the target says nRF RPC libraries, but there is only one library under that page.

Comment (divipillai) [reply doc/nrf/releases/release-notes-changelog.rst:314]: I think we could mention :ref:`nrf_rpc_ipc_readme` instead for :ref:`lib_nrf_rpc`: and refer to as "This library"?

**Review by divipillai — APPROVED** [review]

File: doc/nrf/releases/release-notes-changelog.rst (hunk 4 above)
Comment (umapraseeda) [reply doc/nrf/releases/release-notes-changelog.rst:314]: Yes, I was thinking of this.

**Pull request merged**